plugins: add Debian packaging for PostgreSQL plugin (#723)
bareos/bareos: docs/manuals/source/Appendix/Howtos.rst

> So if you perform your first full backup your are capable to perform differntial and incremental backups.

Reviewer: ```suggestion
After your first full backup you will be able to also run differential and incremental backups.
```
